Penteract/Assets/Scripts/RandomNumberGenerator.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <random>
+
+class RandomNumberGenerator
```

Comment:
This is just a preference, but, wouldn't you like it better to call it RNG, since it is commonly used for random number generators and everyone knows what it is.